src/Alpha_complex/include/gudhi/Alpha_complex.h

```diff
@@ -389,6 +396,9 @@ class Alpha_complex {
     using Simplex_handle = typename SimplicialComplexForAlpha::Simplex_handle;
     using Vector_vertex = std::vector<Vertex_handle>;
 
+    // For users to be able to define their own sqrt function on their desired Filtration_value type
+    using std::sqrt;
```

Comment:
We could perhaps add in the concept of `SimplicialComplexForAlpha` that for "custom" filtration value types, having its own `sqrt` is possible and if not existing `std::sqrt` is used instead?

Reply:
I specified it on 021561a6d